bin/sl-pm.js

```diff
@@ -107,41 +114,67 @@ if (listen == null) {
 mkdirp(base);
 process.chdir(base);
 
-var app = new Server({
-  // Choose driver based on cli options/env once we have alternate drivers.
-  Driver: driver,
-  baseDir: base,
-  basePort: basePort,
-  cmdName: $0,
-  listenPort: listen,
-});
+if (dbDriver === 'sqlite3') {
+  checkAndUpgradeDb(base, function(err) {
+    if (err) {
+      console.error('%s(%d) %s', $0, process.pid, err.message);
+      process.exit(1);
```

Comment:
I know it won't really change the execution, but could this be `return process.exit(1);`?